### PR TITLE
Restore polarSSL

### PR DESCRIPTION
**Pull request by bucanero**

### CODE REVIEW

File: polarssl/PSPBUILD [hunk 1]
```diff
@@ -0,0 +1,35 @@
+pkgname=polarssl
+pkgver=1.3.9
+pkgrel=2
+pkgdesc="Secure Socket Layer library"
+arch=('mips')
+url="https://www.trustedfirmware.org/projects/mbed-tls/"
+license=('GPL2 license')
```
Comment (sharkwouter) [inline polarssl/PSPBUILD:7]: Can just be GPL2
```suggestion
license=('GPL2')
```

**Comment by tpimh** [comment]
What is the reason for restoring this package? From [Wikipedia article](https://en.wikipedia.org/wiki/PolarSSL):

> As of the release of version 1.3.10, PolarSSL has been rebranded to Mbed TLS to better show its fit inside the Mbed ecosystem.
> 
> Starting from version 2.1.0, the library was made available under both the GPL v2 and Apache License v2.0.
> 
> In 2020, Mbed TLS joined the TrustedFirmware project.

I guess, it won't hurt to have multiple versions of the same library if someone needs them.

**Comment by bucanero** [comment]
Already discussed on discord. The API is different, if your code uses polar, you need to migrate everything to the new API. It's useful for compatibility with older apps.

**Comment by sharkwouter** [comment]
As @bucanero said, PolarSSL is not backwards compatible

**Comment by tpimh** [comment]
Makes sense then. I thought it's just a matter of couple of defines to reflect the name change. But if it's more than this, let's just have multiple versions.

**Pull request merged**